lib/src/main/java/com/diffplug/spotless/pom/SortPomStep.java

```diff
@@ -42,7 +42,7 @@ static class State implements Serializable {
 
 		public State(SortPomCfg cfg, Provisioner provisioner) throws IOException {
 			this.cfg = cfg;
-			this.jarState = JarState.from("com.github.ekryd.sortpom:sortpom-sorter:3.0.0", provisioner);
+			this.jarState = JarState.from("com.github.ekryd.sortpom:sortpom-sorter:3.2.1", provisioner);
```

Comment:
You will need to add the missing transitive dependency here

Reply:
Thanks for the reply!

Ah I didn't realize that function accepted a collection of jars.. Do we want that transitive dependency? I only think it's needed for writing the file, and the code change I made completely gets rid of the dependency. I was initially thinking we needed it, but with my changes I don't think it's necessary anymore. 

Your decision though!

Reply:
You shouldn't need to list transitive jars, I was responding to 

> Step 'sortPom' found problem in '':
> org/apache/commons/io/IOUtils
> java.lang.NoClassDefFoundError: org/apache/commons/io/IOUtils

- from https://github.com/diffplug/spotless/discussions/1674

I see now that you fixed it by removing `IOUtils`, it is perfect as-is.

Reply:
Awesome! That's good to hear! Well I hope this can get merged soon, unless you need more changes from me. Glad I could contribute something. 🎉